pkg/controller/control.go

```diff
@@ -52,6 +54,7 @@ type Controller struct {
 }
 
 const (
+	freezeTimeout         = 60 * time.Minute // qemu uses 60 minute timeouts for freezing
```

Comment:
Could you provide the reference link about qemu case you menteiond here?

Reply:
Sure, this is the change that got merged.
https://pve-devel.pve.proxmox.narkive.com/FCClXQF6/rfc-qemu-server-increase-timeout-from-guest-fsfreeze-freeze

Based on the discussion here:
https://forum.proxmox.com/threads/22192/